Cargo.toml

```diff
@@ -50,6 +50,8 @@ tower-http = { version = "0.6.1", features = ["trace"] }
 tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 zeromq = "0.4.0"
+ripemd = "0.1.3"
```

Comment:
Both of these are available already in the `bitcoin` crate, see
https://docs.rs/bitcoin_hashes/0.14.0/bitcoin_hashes/hmac/index.html
https://docs.rs/bitcoin_hashes/0.14.0/bitcoin_hashes/ripemd160/index.html

Reply:
`ripemd160` crate is depricated, they are suggesting to use `ripemd` instead: http://docs.rs/ripemd160

I think it is reasonable to add `hmac` as a separate dependency as well, there isn't any significant cost to doing it.

Reply:
`bitcoin_hashes` does not depend on the `ripemd160` crate
https://crates.io/crates/bitcoin_hashes/0.14.0/dependencies